### PR TITLE
Added 29 missing translations from the todo list

### PR DESCRIPTION
**Pull request by ChristofK**
and changed the translation of the term "Whoops, something went wrong on our servers". The reference to the server was missing in the German translation.

### CODE REVIEW

**Comment by olsza** [comment]
Hay @ChristofK did you make a typo in "Administrator" translation?
Isn't "Administrator" the same in German as in English? As it is, see this information https://github.com/Laravel-Lang/lang/blob/master/CONTRIBUTING.md#what-should-i-do-if-there-is-a-tag-whose-translation-is-the-same-as-in-english

File: json/de.json [hunk 1]
```diff
@@ -1,11 +1,39 @@
 {
     "A fresh verification link has been sent to your email address.": "Ein neuer Bestätigungslink wurde verschickt.",
     "A new verification link has been sent to the email address you provided during registration.": "Ein neuer Bestätigungslink wurde an die E-Mail-Adresse gesendet, die Sie bei der Registrierung angegeben haben.",
+    "Add a new team member to your team, allowing them to collaborate with you.": "Fügen Sie ein neues Teammitglied zu Ihremm Team hinzu und erlauben Sie ihm mit Ihnen zusammenzuarbeiten.",
+    "Add additional security to your account using two factor authentication.":"Fügen Sie Ihrem Konto zusätzliche Sicherheit hinzu, indem Sie die Zwei-Faktor-Authentifizierung verwenden.",
+    "Add Team Member": "Teammitglied hinzufügen",
+    "Add": "Hinzufügen",
+    "Added.": "Hinzugefügt.",
+    "Administrator users can perform any action.": "Administratoren können jede Aktion durchführen.",
+    "Administrator": "Administator",
```
Comment (caouecs) [inline json/de.json:10]: in exclude/de.php

File: json/de.json [hunk 2]
```diff
@@ -1,11 +1,39 @@
...
+    "Are you sure you want to delete your account? Once your account is deleted, all of its resources and data will be permanently deleted. Please enter your password to confirm you would like to permanently delete your account.": "Möchten Sie Ihr Konto wirklich löschen? Sobald Ihr Konto gelöscht wurde, werden alle Ressourcen und Daten dauerhaft gelöscht. Bitte geben Sie Ihr Passwort ein, um zu bestätigen, dass Sie Ihr Konto dauerhaft löschen möchten.",
+    "Are you sure you would like to delete this API token? ": "Möchten Sie dieses API-Token wirklich löschen?",
+    "Are you sure you would like to leave this team?": "Sind Sie sicher, dass Sie dieses Team verlassen möchten?",
+    "Are you sure you would like to remove this person from the team?": "Sind Sie sicher, dass Sie diese Person aus dem Team entfernen möchten?",
     "Before proceeding, please check your email for a verification link.": "Bevor Sie fortfahren, überprüfen Sie bitte Ihr E-Mail-Postfach auf einen Bestätigungslink.",
+    "Browser Sessions": "Browsersitzungen",
     "click here to request another": "klicken Sie hier, um einen neuen anzufordern",
+    "Close": "Schließen",
+    "Code": "Code",
```
Comment (caouecs) [inline json/de.json:27]: in exclude/de.php

**Review by caouecs — CHANGES_REQUESTED** [review]
Danke, just two tags in exclude/de.php

**Comment by ChristofK** [comment]
> Hay @ChristofK did you make a typo in "Administrator" translation?
> Isn't "Administrator" the same in German as in English? As it is, see this information https://github.com/Laravel-Lang/lang/blob/master/CONTRIBUTING.md#what-should-i-do-if-there-is-a-tag-whose-translation-is-the-same-as-in-english
Yes, you are right. It wa just a typo. I removed all strings that do not need to be translated and put them in the exclusion list.

File: json/de.json (hunk 2 above)
Comment (ChristofK) [reply json/de.json]: done

File: json/de.json (hunk 1 above)
Comment (ChristofK) [reply json/de.json]: done

File: json/de.json [hunk 3]
```diff
@@ -7,7 +7,6 @@
     "Add": "Hinzufügen",
     "Added.": "Hinzugefügt.",
     "Administrator users can perform any action.": "Administratoren können jede Aktion durchführen.",
-    "Administrator": "Administator",
```
Comment (olsza) [inline json/de.json]: as for me, you deleted unnecessarily ... you should leave them in their original form ... words that are in "exclusions" files, they only exclude information in the ToDo.md report

File: json/de.json [hunk 4]
```diff
@@ -24,7 +23,6 @@
     "Browser Sessions": "Browsersitzungen",
     "click here to request another": "klicken Sie hier, um einen neuen anzufordern",
     "Close": "Schließen",
-    "Code": "Code",
```
Comment (olsza) [inline json/de.json]: as above

File: json/de.json [hunk 5]
```diff
@@ -34,7 +32,6 @@
     "Create": "Erstellen",
     "Created.": "Erstellt.",
     "Current Password": "Derzeitiges Passwort",
-    "Dashboard": "Dashboard",
```
Comment (olsza) [inline json/de.json:9]: as above

File: json/de.json [hunk 6]
```diff
@@ -53,7 +50,6 @@
     "Invalid signature.": "Ungültige Signatur.",
     "Login": "Anmelden",
     "Logout": "Abmelden",
-    "Name": "Name",
```
Comment (olsza) [inline json/de.json:28]: as above

**Comment by ChristofK** [comment]
I hope everything is right now.

File: json/de.json [hunk 7]
```diff
@@ -1,74 +1,184 @@
...
-    "Forbidden": "Unzulässig",
-    "Forgot your password? No problem. Just let us know your email address and we will email you a password reset link that will allow you to choose a new one.": "Haben Sie Ihr Passwort vergessen? Kein Problem. Teilen Sie uns einfach Ihre E-Mail-Adresse mit und wir senden Ihnen per E-Mail einen Link zum Zurücksetzen des Passworts, über den Sie eine neue auswählen können.",
-    "Forgot your password?": "Passwort vergessen?",
+    "For your security, please confirm your password to continue.": "Um fortzufahren, bestätigen Sie zu Ihrer Sicherheit bitte Ihr Passwort.",
+    "Forbidden": "Verboten",
     "Forgot Your Password?": "Passwort vergessen?",
+    "Forgot your password? No problem. Just let us know your email address and we will email you a password reset link that will allow you to choose a new one.": "Haben Sie Ihr Passwort vergessen? Kein Problem. Teilen Sie uns einfach Ihre E-Mail-Adresse mit und wir senden Ihnen per E-Mail einen Link zum Zurücksetzen des Passworts, über den Sie ein Neues auswählen können.",
+    "Forgot your password?": "Passwort vergessen?",
     "Go Home": "Nach Hause",
+    "Go to page :page": "Gehe zur Seite :page"",
```
Comment (caouecs) [inline json/de.json:58]: Error here with two "

**Review by caouecs — CHANGES_REQUESTED** [review]
Danke but you have a problem on double "

File: json/de.json (hunk 7 above)
Comment (ChristofK) [reply json/de.json]: corrected

**Comment by caouecs** [comment]
Danke

**Pull request merged**